src/main/java/org/primeframework/mvc/security/CipherProvider.java

```diff
@@ -28,7 +28,7 @@
  */
 public interface CipherProvider {
   /**
-   * Return a cipher used for decrypting.
+   * Return a cipher used for decrypting AES/CBC.
```

Comment:
Ideally the interface would not decide the cipher implementation. 
Ideally we would just create a second impl of this interface for GCM, and keep them separate. 

Then the `DefaultEncryptor` would inject both, and start with the GCM impl and fall back.